index.js

```diff
@@ -19,6 +19,10 @@ const Node = require('./node')
 const NODE_TYPES = Node.prototype.types
 const httpMethods = http.METHODS
 
+const versioningStrategies = {
+  ACCEPT_VERSION: require('./versioning-strategies/accept-version')
```

Comment:
Can you just move `versioning-strategies/accept-version` to `lib/accept-version`?

Reply:
done, tnx